lib/libsys/mmap.2

```diff
@@ -113,8 +113,45 @@ Pages may be read.
 Pages may be written.
 .It Dv PROT_EXEC
 Pages may be executed.
+.It Dv PROT_CAP
+CHERI capabilities may be read or written as dictated by
+.Dv PROT_READ
+and
+.Dv PROT_WRITE .
+.It Dv PROT_NO_CAP
+Respect the absence of
+.Dv PROT_CAP .
 .El
 .Pp
+On CHERI platforms, compatability is retained with unmodified POSIX
+programs by implying
+.Dv PROT_CAP
+based on the presence of
+.Dv PROT_READ
+and
+.Dv PROT_WRITE
+respectively, unless the underlying backing store can not safety support
+capabilities (e.g., a
+.Dv MAP_SHARED
+mapping of a file).
+If either of
+.Dv PROT_CAP
+or
+.Dv PROT_NO_CAP
+are set, then capability permissions will not be implied.
```

Comment:
```suggestion
.Dv PROT_CAP
if either of
.Dv PROT_READ
or
.Dv PROT_WRITE
is set unless the underlying backing store can not safely support
capabilities (e.g., a
.Dv MAP_SHARED
mapping of a file).
If
.Dv PROT_NO_CAP
is set, then capability permissions will not be implied.
```

This is specific to squashing down to `PROT_CAP` as then I think the "respectively" language is no longer needed.  Also, I think it's kind of redundant (and possibly confusing) to say that an explicit `PROT_CAP` doesn't imply `PROT_CAP`.